Src/VimApp/VimAppHost.cs

```diff
@@ -14,11 +14,15 @@
 using Vim.Extensions;
 using Vim.Interpreter;
 using Vim.VisualStudio.Specific;
+using System.Collections.Generic;
 
 namespace VimApp
 {
     [Export(typeof(IVimHost))]
+    [Export(typeof(IWpfTextViewCreationListener))]

```

Comment:
Wow that was an oversight. Not sure I want to figure out how long this has been broken.